SimpactApp/templates/SimpactApp/npologin.html

```diff
@@ -31,10 +29,11 @@
 </head>
 
 <body>
-
+	{% block content %}
 	<div class="logincard"> 
 		<div class="row">
-			<form action="/action_page.php" method="post">
+			<form action="/action_page.php" method="POST">
```

Comment:
Not sure about "/action_page.php" - is it a copy-paste leftover artifact?

Should it be replaced with a proper URL?